homeassistant/components/dynalite/bridge.py

```diff
@@ -61,6 +70,27 @@ def update_device(self, device: Optional[DynaliteBaseDevice] = None) -> None:
         else:
             async_dispatcher_send(self.hass, self.update_signal(device))
 
+    @callback
+    def handle_notification(self, notification: DynaliteNotification) -> None:
+        """Handle a notification from the platform and issue events."""
+        if notification.notification == NOTIFICATION_PACKET:
+            self.hass.bus.async_fire(
+                "dynalite_packet",
+                {
+                    CONF_HOST: self.host,
```

Comment:
Please use a constant named `ATTR_HOST`. `CONF_*` constants are only used for config keys.

Reply:
got it. fixing

Reply:
done